infra/terraform/modules/control_panel_api/role.tf

```diff
@@ -134,6 +134,16 @@ resource "aws_iam_policy" "control_panel_api" {
         "arn:aws:iam::${var.account_id}:role/${var.env}_app_*"
       ]
     },
+    {
+      "Sid": "CanListPoliciesForLambdaMigration",
+      "Effect": "Allow",
+      "Action": [
+        "iam:ListPolicies"
+      ],
+      "Resource": [
+        "arn:aws:iam::${var.account_id}:*"
```

Comment:
Allowed to list policies on any resource?  Is that what you meant to do?

Reply:
As far as I know this is to list IAM policies in your account. The action to list policies attached to another IAM resource is different, something like `iam:ListAttachedRolePolicies`.

But in this case I'm interested in listing all the IAM policies in our account, if you [see the migration](https://github.com/ministryofjustice/analytics-platform-control-panel/pull/109/files#diff-7c3ac2e9db789dd8ece232021649f51aR36) it's starting by reading all the IAM policies, etc...